CDP4CrossViewEditor.Tests/Assemblers/CrossviewPMeanArrayAssemblerTestFixture.cs

```diff
@@ -0,0 +1,701 @@
+            (this.elementDefinitions.FirstOrDefault()?.Parameter)?.Remove(P_mean);
+
+            var arrayAssembler = new CrossviewArrayAssembler(this.excelRows, this.parameterTypes);
+
+            // The that power related parameters are present into the header
+            Assert.IsTrue(arrayAssembler.headerDictionary.ContainsKey("redundancy"));
+            Assert.IsTrue(arrayAssembler.headerDictionary.ContainsKey("P_on"));
+            Assert.IsTrue(arrayAssembler.headerDictionary.ContainsKey("P_stby"));
+            Assert.IsTrue(arrayAssembler.headerDictionary.ContainsKey("P_duty_cyc"));
+            Assert.IsFalse(arrayAssembler.headerDictionary.ContainsKey("P_mean"));
+            Assert.AreEqual(13, arrayAssembler.NamesArray.GetLength(1));
+        }
+
+        [Test]
+        public void VerifyThatAssemblerNotCalculatesPMeanIfPDutyCycleAndPeanHaveDifferentStateDependency()
+        {
+            var crossviewSheetRowAssembler = new CrossviewSheetRowAssembler();
+            crossviewSheetRowAssembler.Assemble(this.elementDefinitions);
+            this.excelRows.AddRange(crossviewSheetRowAssembler.ExcelRows);
+
+            Assert.IsNotNull(this.elementDefinitions.FirstOrDefault());
+
+            var P_mean = this.elementDefinitions.FirstOrDefault()?.Parameter.FirstOrDefault(p => p.ParameterType.ShortName == "P_mean");
+
+            if (P_mean != null)
+            {
+                P_mean.StateDependence = null;
+            }
+
+            var arrayAssembler = new CrossviewArrayAssembler(this.excelRows, this.parameterTypes);
+
+            // The that power related parameters are present into the header
+            Assert.IsTrue(arrayAssembler.headerDictionary.ContainsKey("redundancy"));
+            Assert.IsTrue(arrayAssembler.headerDictionary.ContainsKey("P_stby"));
+            Assert.IsTrue(arrayAssembler.headerDictionary.ContainsKey("P_on"));
+            Assert.IsTrue(arrayAssembler.headerDictionary.ContainsKey("P_duty_cyc"));
+            Assert.IsTrue(arrayAssembler.headerDictionary.ContainsKey("P_mean"));
+        }
+
+        [Test]
+        public void VerifyThatAssemblerNotCalculatesPMeanIfPDutyCycleAndPMeanHaveDifferentOptionDependency()
+        {
+            var crossviewSheetRowAssembler = new CrossviewSheetRowAssembler();
+            crossviewSheetRowAssembler.Assemble(this.elementDefinitions);
+            this.excelRows.AddRange(crossviewSheetRowAssembler.ExcelRows);
+
+            Assert.IsNotNull(this.elementDefinitions.FirstOrDefault());
+
+            var P_duty_cyc = this.elementDefinitions.FirstOrDefault()?.Parameter.FirstOrDefault(p => p.ParameterType.ShortName == "P_duty_cyc");
+
+            if (P_duty_cyc != null)
+            {
+                P_duty_cyc.IsOptionDependent = true;
+            }
+
+            var arrayAssembler = new CrossviewArrayAssembler(this.excelRows, this.parameterTypes);
+
+            // The that power related parameters are present into the header
```

Comment:
Typo in comment "the that power related..."?

Given that this is repeated to many times, I would just remove this comment completely.